dev/src/document.ts

```diff
@@ -626,7 +647,7 @@ export class DocumentMask {
    * @returns {DocumentMask}
    */
   static fromUpdateMap(data) {
-    let fieldPaths = [];
+    const fieldPaths: FieldPath[] = [];
```

Comment:
Method signature: `fromUpdateMap(data:UpdateData) : DocumentMask`

Reply:
The plan for types (here and beyond) is to come back around and do that in another PR.  I like to do the bare minimum to get a file converted to TypeScript, then make it type-awesome in a followup :)